control/control.rnc

```diff
@@ -213,6 +215,10 @@ self_update_id =			element self_update_id { text }
 # Installation UI (boo#1088785)
 ## Enables alternative look of installation with sidebar
 installation_ui =			element installation_ui { "sidebar" }
+# Media name which contains ALL packages for an useful installation
+full_system_media_name =		element full_system_media_name { text }
+# URL where the media described in full_system_download_url is located.
```

Comment:
Ah, I forgot that we should mention that both `full_system_media_name` and `full_system_download_url` must be defined.

I think we should add this at least to `master`.